blocks/melody/ff_module/ff_module-button/ff_module-button.less

```diff
@@ -6,6 +6,7 @@
         border: 1px solid @ff_colour_default;
         padding: (@ff_size_spacing_min * .85) @ff_size_spacing_med;
         cursor: pointer;
+        white-space: nowrap;
```

Comment:
This is to ensure that text on the button remains on one line, despite pressure from any containing elements. @aegirfirefly can you please confirm this is as designed?